cmd/exec_test.go

```diff
@@ -321,15 +323,32 @@ main contains "hello" if {
 
 					ctx, cancel := context.WithCancel(context.Background())
 					defer cancel()
-					go func() {
+					go func(expectedErrors []string) {
```

Comment:
I'm trying to understand what we're testing here?

Reply:
the existing test is verifying that the errors are present in the status, 
my goal was to also verify that these errors are also returned directly from the trigger method
handing over the expected error as a parameter became necessary due to a detected race condition as the goroutine will otherwise work on `tc.expErrs` which is changed concurrently in the outer loop (l. 294)